books/free-programming-books-zh.md

```diff
@@ -152,12 +150,7 @@
 * [The Linux Command Line](http://billie66.github.io/TLCL/index.html)
 * [Ubuntu 参考手册](http://wiki.ubuntu.org.cn/UbuntuManual)
 * [uCore Lab: Operating System Course in Tsinghua University](https://www.gitbook.com/book/objectkuan/ucore-docs/details)
-* [UNIX TOOLBOX](http://cb.vu/unixtoolbox_zh_CN.xhtml)
-
-
-### 程序员杂谈
-
-* [程序员的自我修养](http://www.kancloud.cn/kancloud/a-programmer-prepares)
```

Comment:
If it works... why remove it?

Reply:
I don't know why I deleted it, I will add it back.

The resource on kancloud however links the one on gitbooks so to avoid problems I would leave it that way.